dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -203,36 +200,27 @@
 			status = "disabled";
 		};
 
+#define K_PIN_DEFINE(k_name, k_port, k_pin, k_mux) \
+	k_name##_##pt##k_port##k_pin: k_name##_##pt##k_port##k_pin { \
+		pin = <k_pin>; \
+		mux = <k_mux>; \
+	}
```

Comment:
Think a `pinmux` property like mentioned in `Documentation/devicetree/bindings/pinctrl/pinctrl-bindings.txt` could be used here?